src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -175,6 +175,19 @@ public Double FontSize
             }
         }
 
+        [JsonIgnore]
```

Comment:
Can you add some comments to this public property?

Reply:
Done.